lib/typeform_data/requestor.rb

```diff
@@ -11,6 +11,11 @@ module Requestor
 
       Errno::ECONNREFUSED,
       TypeformData::TransientResponseError,
+
+      # Sometimes it takes a while before Typeform's state becomes consistent. In particular, this
+      # can be an issue if you receive a webhook for a form response, then immediately request that
+      # response from Typeform's servers.
+      TypeformData::InvalidEndpointOrMissingResource
```

Comment:
Agreed -- ideally, I think we would only retry on calls originating from `#responses`.

Yes I think a better solution is out of scope for now, but I will add the docs and an issue.